go/flagutil/flagutil.go

```diff
@@ -124,3 +125,58 @@ func (value StringMapValue) String() string {
 	sort.Strings(parts)
 	return strings.Join(parts, ",")
 }
+
+// DualCaseStringListVar cretes a flag which supports both dashes and underscores
```

Comment:
typo: cretes -> creates (applies to all the functions)

Reply:
fixed.